### PR TITLE
fixing slug in Link for "About Us"

### PR DESCRIPTION
**Pull request by Stevensauro**

### CODE REVIEW

File: fetools-app/src/pages/CharacterFinder.jsx [hunk 1]
```diff
@@ -2,9 +2,9 @@ import '../index.css';
 
 function CharacterFinder() {
     return (
-      <div>
+      <main>
```
Comment (joekotvas) [inline fetools-app/src/pages/CharacterFinder.jsx:5]: Love the semantic elements!

File: fetools-app/src/components/GoDeeper.jsx
Comment (joekotvas) [inline fetools-app/src/components/GoDeeper.jsx:1]: Great work here! I've added a PR with a suggested refactor for this file. No major changes, but a few things that can hopefully make it more readable and succinct. Also changed the h1 to an h2.

Please check it out!

https://github.com/chingu-voyages/v47-tier2-team-17/pull/18

**Review by joekotvas — APPROVED** [review]
Good work, Steven. I've added a PR with a suggested refactor for GoDeeper.jsx 

Please look it over and if you are good with it merge it into your branch before merging yours into dev.
https://github.com/chingu-voyages/v47-tier2-team-17/pull/18

File: fetools-app/src/components/GoDeeper.jsx
Comment (Stevensauro) [reply fetools-app/src/components/GoDeeper.jsx:1]: Thanks Joe, I saw that haha, I'm going to take a look into your refactoring so I can code more efficiently!

**Pull request merged**